pkg/components/exec/file.go

```diff
@@ -29,9 +29,10 @@ type FileInfo struct {
 }
 
 const (
-	envServiceName   = "OTEL_SERVICE_NAME"
```

Comment:
Ignore this change and file_test.go, I'll merge with main once that PR is merged. I needed it in my branch for local testing and it ended up here.